client/pages/prs.jsx

```diff
@@ -0,0 +1,287 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class PrPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      prs: [],
+      userId: 1
```

Comment:
Rather than hard-code the user id on the front end, just hard-code it on the back end.